src/FSharp.Data.GraphQL.Server.Middlewares/TypeSystemExtensions.fs

```diff
@@ -0,0 +1,16 @@
+namespace FSharp.Data.GraphQL.Server.Middlewares
+
+open FSharp.Data.GraphQL.Types
+
+[<AutoOpen>]
+module TypeSystemExtensions =
```

Comment:
This module is meant to add extensions to add metadata used by the QueryWeightMiddleware.